images/image.go

```diff
@@ -388,13 +407,32 @@ func RootFS(ctx context.Context, provider content.Provider, configDesc ocispec.D
 	return config.RootFS.DiffIDs, nil
 }
 
+// IsEncryptedDiff returns true if mediaType is a known encrypted media type.
```

Comment:
Can you move these functions to separate file? Maybe a different package in the future but should keep the files separated from the main image functions.